code/game/objects/structures/crates_lockers/closets/secure/bar.dm

```diff
@@ -23,3 +23,26 @@
 	new /obj/item/reagent_containers/food/drinks/bottle/small/beer( src )
 	new /obj/item/reagent_containers/food/drinks/bottle/small/beer( src )
 	new /obj/item/reagent_containers/food/drinks/bottle/small/beer( src )
+
+/obj/structure/closet/secure_closet/cabinet/beer
+	name = "beer closet"
+	desc = "A cabinet stacked end-to-end with six packs of beer."
+	req_access = null
+	storage_capacity = 45
+
+/obj/structure/closet/secure_closet/cabinet/beer/fill()
+	new /obj/item/storage/box/fancy/yoke/beer( src )
+	new /obj/item/storage/box/fancy/yoke/beer( src )
+	new /obj/item/storage/box/fancy/yoke/ebisu( src )
+	new /obj/item/storage/box/fancy/yoke/ebisu( src )
+	new /obj/item/storage/box/fancy/yoke/shimauma( src )
+	new /obj/item/storage/box/fancy/yoke/shimauma( src )
+	new /obj/item/storage/box/fancy/yoke/moonlabor( src )
+	new /obj/item/storage/box/fancy/yoke/moonlabor( src )
+	new /obj/item/storage/box/fancy/yoke/earthmover( src )
+	new /obj/item/storage/box/fancy/yoke/earthmover( src )
+	new /obj/item/storage/box/fancy/yoke/whistlingforest( src )
+	new /obj/item/storage/box/fancy/yoke/whistlingforest( src )
```

Comment:
```suggestion
	new /obj/item/storage/box/fancy/yoke/beer(src)
	new /obj/item/storage/box/fancy/yoke/beer(src)
	new /obj/item/storage/box/fancy/yoke/ebisu(src)
	new /obj/item/storage/box/fancy/yoke/ebisu(src)
	new /obj/item/storage/box/fancy/yoke/shimauma(src)
	new /obj/item/storage/box/fancy/yoke/shimauma(src)
	new /obj/item/storage/box/fancy/yoke/moonlabor(src)
	new /obj/item/storage/box/fancy/yoke/moonlabor(src)
	new /obj/item/storage/box/fancy/yoke/earthmover(src)
	new /obj/item/storage/box/fancy/yoke/earthmover(src)
	new /obj/item/storage/box/fancy/yoke/whistlingforest(src)
	new /obj/item/storage/box/fancy/yoke/whistlingforest(src)
```
i assume that you followed the booze closet example. if you could fix that as well i'd appreciate it, but you don't have to of course.